src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.module.css

```
@@ -0,0 +1,101 @@
+/* Outter Grid */
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 30px 1fr 30px;
+    grid-template-rows: auto;
+  }
+}
+
+@media (--tablet-and-down) {
+  .bleedBackground {
+    grid-row: 1;
+    grid-template-rows: auto;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-column-start: 2;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding: 100px 0;
+  row-gap: 50px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+.gridAreaMiddle {
+  grid-column: 1 / span 12;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    column-gap: 10px;
+    grid-column: 1/ span 13;
+    grid-template-columns: repeat(13, minmax(0, 1fr));
+    padding: 0;
+  }
+
+  .gridAreaMiddle {
+    grid-column: 1 / span 13;
+  }
+}
+
+
+.innerWrapper {
+  background: var(--color-gray-300);
+  height: 100%;
+  margin: 0;
+  padding: 60px 50px;
+}
+ 
+@media (--tablet-and-down) {
+  .innerWrapper {
+    grid-column: 1 /span 13;
+    grid-row: 1;
+  }
+} 
+
+.quote {
+  font: var(--font-body-medium);
+  margin: 0;
+}
+
+.attribution {
+  font: var(--font-caption);
+  margin-top: 22px;
+}
+
+.dots {
+  padding-top: 19px;
+  text-align: center;
+}
+
+.dots button {
```

It's probably not a good idea to use a tag name selector like this, since you're making certain assumptions about the internals of the carousel library's implementation.

One way that we may be able to select the individual buttons without using a tag selector would be to mix both local and global class names in this selector. This is a bit of an advanced usage of CSS modules, but you can [mark some classes as `:global`](https://github.com/css-modules/css-modules#exceptions), which tells CSS modules to not automatically generate a unique name anymore.

Since the built-in class name for the react-pure-carousel's `<Dot/>` component is named `carousel__dot`, I think we could replace this selector with:

```suggestion
.dots :global(.carousel__dot) {
```

I haven't tried this out before, so I'm not sure if this is exactly right, but I think it'll work out something like that.

---

👍 Changed.